llarp/ip.cpp

```diff
@@ -16,11 +16,7 @@ namespace llarp
     bool
     IPv4Packet::Load(llarp_buffer_t pkt)
     {
-#ifndef MIN
-#define MIN(a, b) (a < b ? a : b)
-      sz = MIN(pkt.sz, sizeof(buf));
-#undef MIN
-#endif
+      sz = std::min(pkt.sz, sizeof(buf));
```

Comment:
I wonder if this is going to cause some cross platform problems (I think we tried this before)

Reply:
the windows compiler usually requires `#include <algorithm>` *shrugs*]

Reply:
Initial code was stupid because it wouldn't assign anything at to `sz` all if `MIN` was defined.
I'll add <algorithm>, thanks for mentioning that.

Reply:
Ah I see, yea good catch. Just thought it was a simple swap.